docs/content/walkthrough.md

```diff
@@ -69,18 +66,17 @@ INFO[0000] Starting application at 127.0.0.1:3000
 
 Note: [Building Athens Go Modules enabled is not yet supported](https://github.com/gomods/athens/pull/371), so we have turned it off in the above example.
 
-<!--
 **PowerShell**
 ```console
 $ mkdir $(go env GOPATH)\src\github.com\gomods
 $ cd $(go env GOPATH)\src\github.com\gomods
 $ git clone https://github.com/gomods/athens.git
 $ cd athens
+$ $env:GO111MODULE = "off"
 $ start -NoNewWindow go "run .\cmd\proxy"
```

Comment:
This should be `Start-Process -NoNewWindow go run .\cmd\proxy`

Reply:
You're right. Better to use `Start-Process` than the alias.